ui/src/App.tsx

```diff
@@ -102,6 +110,17 @@ export function App() {
           onClick={handleNavigate(params.row)}
           disabled={actionInProgress}
         />,
+        <GridActionsCellItem
```

Comment:
the list of actions is growing quite a lot, don't you think a [3 dots menu](https://mui.com/material-ui/react-menu/) would be better?

Reply:
I've moved some actions to the 3 dots menu, thanks!